notifications_api_common/admin.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib import admin, messages
+from django.utils.translation import gettext_lazy as _
+
+from requests.exceptions import RequestException
+from solo.admin import SingletonModelAdmin
+from zds_client.client import ClientError
+
+from .models import NotificationsConfig, Subscription
+
+
+@admin.register(NotificationsConfig)
+class NotificationsConfigAdmin(SingletonModelAdmin):
+    list_display = (
+        "notifications_api_service",
+        "subscriptions",
```

Comment:
I don't really see much added value here, since we have the `Subscription` model anyway?

It probably doesn't even work because of the dropped FK field :-)

Besides that - `list_display` is not used in `SingletonModelAdmin` afaik, so the whole config should probably be replaced with just a `pass` statement.